docs/server-side/v2/api-reference/Rooms/update-a-room.mdx

```diff
@@ -76,9 +78,13 @@ curl --location --request POST 'https://api.100ms.live/v2/rooms/<room_id>' \
+| description <br/>          | `string`  | A string to describe your room's usage. For example "9PM English Class Batch 2"                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | No       |
+| recording_info <br/>       | `object`  | Object of type `recording_info`. This object contains information for enabling recording/setting storage location for recordings. <br/><br/>Check the [recording_info arguments](/server-side/v2/Rooms/create-via-api#recording-info-arguments) below for more information. <br/><br/> **Note:** This argument is only applicable to enable/disable [SFU recording](/server-side/v2/Destinations/recording). Refer to [RTMP Streaming & Browser Recording](/server-side/v2/Destinations/rtmp-streaming-and-browser-recording) guide for other options. | No       |
+| region <br/>               | `string`  | Region in which you want to create a room. <br/><br/> `eu` - European Union <br/><br/> `in` - India <br/><br/> `us` - United States <br/><br/><br/> `auto` Fallback to region of the template (default value)                                                                                                                                        | No       |
+| large_room <br/>           | `boolean` | Enable this flag if the requirement is to support more than 2500 peers in the room. You are also required to update to latest SDKs to support this functionality. By default, this value is set to `false`. Learn more about it in the [large room creation guide](/server-side/v2/how-to-guides/create-large-rooms).                                          | No       |                                                                                                             | No       |
+| size <br/>                 | `int`     | This value is used to define the maximum number of peers expected to be joining the given room. Following are the acceptable ranges: <br/> <br/>  With `large_room: false`, Minimum `size` = 0; Maximum `size` = 2500  <br/> <br/>  With `large_room: true`, Minimum `size` = 0; Maximum `size` = 20000                                                        | No       |
+| max_duration_seconds <br/> | `int`     | This value is used to define the maximum duration in seconds for a room session. Following are the acceptable ranges: <br/> Minimum `max_duration_seconds` = 120 <br/> Maximum `max_duration_seconds` = 86100  <br/> <br/>                                                                                                                                     | No       |
```

Comment:
no defaults ?

Reply:
default is nil. It's not returned if not set in the room

Reply:
Resolving this for now.